.mage/magefile.go

```diff
@@ -41,14 +28,38 @@ func init() {
 
 func All() {
 	mg.Deps(
-		mg.F(mgconvco.ConvcoCheck, "origin/main..HEAD"),
-		mggolangcilint.GolangciLint,
-		mggoreview.Goreview,
-		mgmarkdownfmt.FormatMarkdown,
-		mgyamlfmt.FormatYaml,
+		ConvcoCheck,
+		GolangciLint,
+		Goreview,
+		FormatMarkdown,
+		FormatYaml,
 	)
 	mg.SerialDeps(
-		mggo.GoModTidy,
+		GoModtidy,
 		mggitverifynodiff.GitVerifyNoDiff,
 	)
 }
+
+func FormatYaml() error {
```

Comment:
Are these functions added for removing the magic `// mage:import` ?

in any case, an added bonus is that you don't need to specify an argument when running only e.g. convco-check

Reply:
With the new structure for running commands centered around `exec.Cmd`, most of the public functions in the tools packages are no longer valid mage targets, so to keep the mental model simple we're thinking that we should instead avoiding having ready-made mage-targets altogether and instead write those in the `magefile.go`.

I think this will lead to a simpler mental model, since there will be a 1-1 mapping between Make targets and Go functions in `magefile.go`.

In short, yes, no more `//mage:import`!

Reply:
Nice, I like it!

Reply:
@viktorvoltaire Minor thing, but we should rename the `targets` folder to `tools` - since it now contains tools and not targets. :smile_cat: